### ISSUE TITLE
Chrome IOS not detected

### ISSUE THREAD

**Issue by neufman** [Wolfcast/BrowserDetection#24]
Hello,

It seems that chrome ios is not detected.
Below 2 different examples on 2 iphone :

```
object(BrowserDetection)#51 (12) {
 ["_agent":"BrowserDetection":private]=>
 string(142) "Mozilla/5.0 (iPhone; CPU iPhone OS 11_4_1 like Mac OS X) AppleWebKit/604.1.34 (KHTML, like Gecko) CriOS/68.0.3440.83 Mobile/15G77 Safari/604.1"
 ["_aolVersion":"BrowserDetection":private]=>
 string(0) ""
 ["_browserName":"BrowserDetection":private]=>
 string(6) "Safari"
 ["_compatibilityViewName":"BrowserDetection":private]=>
 string(0) ""
 ["_compatibilityViewVer":"BrowserDetection":private]=>
 string(0) ""
 ["_is64bit":"BrowserDetection":private]=>
 bool(false)
 ["_isAol":"BrowserDetection":private]=>
 bool(false)
 ["_isMobile":"BrowserDetection":private]=>
 bool(true)
 ["_isRobot":"BrowserDetection":private]=>
 bool(false)
 ["_platform":"BrowserDetection":private]=>
 string(6) "iPhone"
 ["_platformVersion":"BrowserDetection":private]=>
 string(7) "unknown"
 ["_version":"BrowserDetection":private]=>
 string(7) "unknown"
}
```
```
object(Wolfcast\BrowserDetection)#51 (12) {
  ["_agent":"Wolfcast\BrowserDetection":private]=>
  string(118) "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_10_4) AppleWebKit/600.7.12 (KHTML, like Gecko) Version/8.0.7 Safari/600.7.12"
  ["_browserName":"Wolfcast\BrowserDetection":private]=>
  string(6) "Safari"
  ["_compatibilityViewName":"Wolfcast\BrowserDetection":private]=>
  string(0) ""
  ["_compatibilityViewVer":"Wolfcast\BrowserDetection":private]=>
  string(0) ""
  ["_customBrowserDetection":"Wolfcast\BrowserDetection":private]=>
  array(0) {
  }
  ["_customPlatformDetection":"Wolfcast\BrowserDetection":private]=>
  array(0) {
  }
  ["_is64bit":"Wolfcast\BrowserDetection":private]=>
  bool(false)
  ["_isMobile":"Wolfcast\BrowserDetection":private]=>
  bool(false)
  ["_isRobot":"Wolfcast\BrowserDetection":private]=>
  bool(false)
  ["_platform":"Wolfcast\BrowserDetection":private]=>
  string(9) "Macintosh"
  ["_platformVersion":"Wolfcast\BrowserDetection":private]=>
  string(7) "10.10.4"
  ["_version":"Wolfcast\BrowserDetection":private]=>
  string(5) "8.0.7"
}
```
Thanks for your help in advance !

**Comment by Wolfcast** [comment]
It's weird that the user-agent does not contain "Chrome/###" at all. Is that a "real" Chrome browser? What is the device model? OS installed?

**Comment by neufman** [comment]
iPhone 7 Plus, iOS 11.4.1 and iPhone 6s , iOS 11.4.1 with app [Google Chrome](https://itunes.apple.com/fr/app/google-chrome/id535886823?mt=8)

**Comment by Wolfcast** [comment]
Thanks it's a bug in the library which applies for Chrome on iOS. I will fix this today or tomorrow!

**Comment by Wolfcast** [comment]
Fixed with Version 2.9.1 which is out now! Your second example seems to be from Safari 8.0.7 under a desktop using OS X Yosemite and not an iPhone... Are you sure you sent the correct User-Agent?

**Comment by neufman** [comment]
Yes I'm sure :)
See the capture in the attached file.

![file](https://user-images.githubusercontent.com/7502193/44570416-0ec04680-a77e-11e8-9f31-5d1aa69f9615.png)
